pydocmd.yml

```diff
@@ -1,6 +1,6 @@
 site_name: "Gitcoin Developer Documentation"
 repo_name: 'gitcoinco/web'
-repo_url: 'https://github.com/gitcoinco/web'
```

Comment:
what does removing the `repo_url` do here?

Reply:
@danlipert MkDocs automatically depending upon the theme takes this parameter `repo_url` and creates an edit link over every page.

Reply:
@agbilotia1998 I see - does removing the `repo_url` affect anything else in the docs?

Reply:
@danlipert seems nothing else gets effected.

Reply:
@agbilotia1998 great - how did you test that nothing else was affected?

Reply:
A basic sanity for the redirections to pages and check if the url is present at other places. May be you can apply this patch and check if I've missed anything.

Reply:
A place where additionally this repo url is present is at the header where the number of stars and forks for the repo are present. This gets removed on deleting the repo url.

Reply:
can't we set the edit_uri to '' and leave the repo url ?

Reply:
@thelostone-mc some themes take the repo url and automatically checks hosts like github.com, if present, it automatically creates an edit link by appending `/edit/master/docs` to the repo_url for every page.

Reply:
@agbilotia1998 would be good to keep the stars and forks info.. is there some way we can disable the edit links without removing the metrics?

Reply:
@danlipert @thelostone-mc this PR in pydocmd package fixes the problem by adding the `edit_uri` parameter in the mkdocs configuration https://github.com/NiklasRosenstein/pydoc-markdown/pull/80
This change will be included in v2.0.6 which is not released yet, currently, we are using v2.0.4 and v2.0.5 also does not provide a fix for this. For now, maybe we can just comment the repo_url parameter stating to be used with v2.0.6 or other thing that can be done is cloning this package and adding `edit_uri` parameter at https://github.com/NiklasRosenstein/pydoc-markdown/blob/54c33c8845e5936a25667ebb20d10a9913793910/pydocmd/__main__.py#L75

Reply:
Hmm... looks like v2.0.5. was released over a year ago... in that case I'm fine with having the metrics removed for now since it doesn't look like v2.0.6 is coming any time soon.